src/main/scala/com/campudus/tableaux/database/model/structure/TableModel.scala

```diff
@@ -45,13 +46,18 @@ class TableModel(val connection: DatabaseConnection)(
 
           (t, result) <- t
             .query(
-              s"INSERT INTO system_table (user_table_name, is_hidden, langtags, type, group_id) VALUES (?, ?, ?, ?, ?) RETURNING table_id",
+              s"INSERT INTO system_table (user_table_name, is_hidden, langtags, type, group_id, attributes) VALUES (?, ?, ?, ?, ?, ?) RETURNING table_id",
               Json
                 .arr(name,
                      hidden,
                      langtags.flatMap(_.map(f => Json.arr(f: _*))).orNull,
                      tableType.NAME,
-                     tableGroupIdOpt.orNull)
+                     tableGroupIdOpt.orNull,
+                     attributes match {
+                       case Some(obj) => obj.encode()
+                       case None => "{}"
+                     }
+                     )
```

Comment:
[Minor] Wrong indentation. I would suggest to place both brackets in one line like `})`